cms-contentful/README.md

```diff
@@ -0,0 +1,22 @@
+Sunrise Java Contentful CMS
+===========================
+
+Module for [Sunrise Java](https://github.com/sphereio/commercetools-sunrise-java) with Contentful service. 
+
+### How to use it
+
+To implement Contentful data into sunrise project, 
+one need to create ContentfulCmsService object using API token and space id:
+
+`ContentfulCmsService cmsService = ContentfulCmsService.of("spaceId", "token");`
+
+It will automatically fetch all entries.
```

Comment:
It is true that we want to cache it and we have to find the best way to do it, but all of it at the beginning is not a good idea, because of what @schleichardt said. I think it would make more sense to build the cache in accordance to the requested data.
1. So at the beginning the cache is empty
2. When you request a field you check if you have it in the cache
3a. If you have it, then you load the info from the cache.
3b. If you don't have it, then you fetch that content and cache it.

The question would remain on how to invalidate the content. It can automatically invalidate stale content (according to some rule) or it can be cleared out manually. The first one would be more elegant, but we have to find a good algorithm or it can backfire.

But for the 1st PR, as I told you, just focus on getting the requested field, without the caching part.